### PR TITLE
Add a `Name` header to inspected Class objects

### PR DESCRIPTION
**Pull request by vemv**
> Closes https://github.com/clojure-emacs/orchard/issues/217

Adds a `Name` header to inspected Class objects.

Also makes some tests less fragile - they were breaking on JDK 21.

Cheers - V

### CODE REVIEW

File: src/orchard/inspect.clj [hunk 1]
```diff
@@ -686,7 +689,7 @@
 
 (defn- render-section [obj inspector [section sort-key-fn]]
   (let [method (symbol (str ".get" (name section)))
-        elements (eval (list method obj))]
+        elements (eval (list method obj))] ;; XXX ditch eval
```
Comment (bbatsov) [inline src/orchard/inspect.clj:692]: XXX? Should be `TODO:` or `FIXME:`?

**Comment by bbatsov** [comment]
I'm kind of puzzled what exactly is the difference between "Name" and "Class", though, as it seem the name is derived from the class.

**Comment by vemv** [comment]
If we're inspecting the class object `java.lang.String`, its name is `java.lang.String` and its class is `java.lang.Class`

**Comment by bbatsov** [comment]
But inspecting classes will be the only case in which those 2 are different, right? I'm just wondering if "Name" is the right name for this.

**Comment by vemv** [comment]
Yes, only `defmethod inspect :class` is affected

File: src/orchard/inspect.clj (hunk 1 above)
Comment (vemv) [reply src/orchard/inspect.clj]: > XXX? Should be TODO: or FIXME:?

fixed by actually addressing the thing 😄

**Pull request merged**